psse/psse-converter/src/main/java/com/powsybl/psse/converter/TransformerConverter.java

```diff
@@ -0,0 +1,794 @@
+        double rw;
+        double xw;
+        switch (cz) {
+            case 1:
+                rw = r;
+                xw = x;
+                break;
+            case 2:
+                // change to right Sbase pu
+                rw = r * sbase / windingSbase;
+                xw = x * sbase / windingSbase;
+                break;
+            case 3:
+                // convert load loss power and current into pu impedances
+                rw = r / windingSbase / 1000000;
+                xw = Math.sqrt(x * x - rw * rw);
+
+                rw = rw * sbase / windingSbase;
+                xw = xw * sbase / windingSbase;
+                break;
+            default:
+                throw new PsseException("PSSE: Unexpected CZ = " + cz);
+        }
+        return new Complex(rw, xw);
+    }
+
+    private static Complex defineShuntAdmittance(String id, double magG, double magB, double sbase, double windingSbase,
+        double baskv, double nomV, int cm) {
+        double g;
+        double b;
+        switch (cm) {
+            case 1:
+                // g and b represent the values of the magnetizing admittance at the i end in pu at 1/Zb1 base where Zb1 = Vb1*Vb1/Sb1
+                // Vb1 is the bus i voltage base (BASKV) and Sb1 is the system MVA base which is SBASE
+                g = magG;
+                b = magB;
+                break;
+            case 2:
+                g = magG / (1000000 * sbase) * (baskv / nomV) * (baskv / nomV);
+                double y = magB * (windingSbase / sbase) * (baskv / nomV) * (baskv / nomV);
+                double b2 = y * y - g * g;
+                if (b2 >= 0) {
+                    b = -Math.sqrt(b2);
+                } else {
+                    b = 0.0;
+                    LOGGER.warn("Magnetizing susceptance of Transformer ({}) set to 0 because admittance module is ({}) and conductance is ({})  ", id, y, g);
+                }
+                break;
+            default:
+                throw new PsseException("PSSE: Unexpected CM = " + cm);
+        }
+        return new Complex(g, b);
+    }
+
+    private static ComplexRatio defineComplexRatio(double windV, double ang, double baskv, double nomV, int cw) {
+        return new ComplexRatio(defineRatio(windV, baskv, nomV, cw), defineAngle(ang));
+    }
+
+    private static double defineRatio(double windV, double baskv, double nomV, int cw) {
+        double ratio;
+        switch (cw) {
+            case 1:
+                ratio = windV;
+                break;
+            case 2:
+                ratio = windV / baskv;
+                break;
+            case 3:
+                ratio = windV * nomV / baskv;
+                break;
+            default:
+                throw new PsseException("PSSE: Unexpected CW = " + cw);
+        }
+        return ratio;
+    }
+
+    private static double defineAngle(double ang) {
+        return ang;
+    }
```

Comment:
Is this method necessary?

Reply:
Deleted. I kept emphasizing that there is not conversion associated to the angle